### PR TITLE
added username to User model, added username field to registration pr…

### PR DESCRIPTION
**Pull request by fordfischer**
…ocess, modified user factory to include a username for testing

### CODE REVIEW

File: db/migrate/20151124223641_add_user_name_to_users.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class AddUserNameToUsers < ActiveRecord::Migration
+  def change
+  end
```
Comment (lasley) [inline db/migrate/20151124223641_add_user_name_to_users.rb]: Should there be a column creation here?

**Comment by lasley** [comment]
Hi @fordfischer - This branch is now in conflict and will need resolution. Take a look at http://githowto.com/resolving_conflicts for info on how to resolve.

**Comment by fordfischer** [comment]
@lasley is this better? I added a column in the migration and resolved the conflicts

**Comment by lasley** [comment]
:+1:

**Pull request merged**